io/plugins/eu.esdihumboldt.hale.io.xls.ui/src/eu/esdihumboldt/hale/io/xls/ui/XLSInstanceExportConfigurationPage.java

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2023 wetransform GmbH
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     wetransform GmbH <http://www.wetransform.to>
+ */
+
+package eu.esdihumboldt.hale.io.xls.ui;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import javax.xml.namespace.QName;
+
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.jface.viewers.ArrayContentProvider;
+import org.eclipse.jface.viewers.CheckStateChangedEvent;
+import org.eclipse.jface.viewers.CheckboxTableViewer;
+import org.eclipse.jface.viewers.ICheckStateListener;
+import org.eclipse.jface.viewers.LabelProvider;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Group;
+import org.eclipse.swt.widgets.Table;
+import org.eclipse.ui.PlatformUI;
+
+import eu.esdihumboldt.hale.common.core.io.Value;
+import eu.esdihumboldt.hale.common.instance.io.InstanceWriter;
+import eu.esdihumboldt.hale.common.schema.model.TypeDefinition;
+import eu.esdihumboldt.hale.io.csv.InstanceTableIOConstants;
+import eu.esdihumboldt.hale.io.csv.ui.InstanceExportConfigurationPage;
+
+/**
+ * Configuration page for exporting Excel
+ * 
+ * @author Emanuela Epure
+ */
+public class XLSInstanceExportConfigurationPage extends InstanceExportConfigurationPage {
```

Comment:
Why extend the CSV page?
To me it seems you overwrite all logic defined in that page without calling the super implementation, so basically nothing from the super class is actually used?

Reply:
At this moment I'm just using 2 buttons name from the top page, but as no super() is used I could extent the page to AbstractConfigurationPage<InstanceWriter, IOWizard<InstanceWriter>> and have the buttons inside XLSInstanceExportConfigurationPage.

Reply:
I think just for reusing field declarations it does not make sense to extend the other class.
If there is similar functionality you could however see if there is potential for sharing the logic between the two classes, e.g. by moving the common functionality to a shared method that can be used by both.